ios/MaasNearbyTransit/Sources/MaasNearbyTransit/Components/NearbyTransitStopListItem.swift

```diff
@@ -0,0 +1,129 @@
+import SwiftUI
+import Swappable
+import MaasComponents
+
+public struct NearbyTransitStopListItem: View, Swappable {
+
+    @Environment(\.textStyleTextL) var textStyleTextL
+    @Environment(\.textStyleTextM) var textStyleTextM
+    @Environment(\.grayScale) var grayScale
+
+    public struct InputType {
+        public let data: StopWithSchedulesWithDepartures
+    }
+    public let input: InputType
+
+    public init(input: InputType) {
+        self.input = input
+    }
+
+    public var defaultBody: some View {
+        Cell(
+            content: { content },
+            prefix: { prefix }
+        )
+    }
+}
+
+// MARK: - Content
+
+private extension NearbyTransitStopListItem {
+
+    var content: some View {
+        VStack(alignment: .leading, spacing: Spacing.xs.value) {
+
+            VStack(alignment: .leading, spacing: Spacing.xxs.value) {
+                name
+                HStack(spacing: 0) {
+                    walkDuration
+                    direction
+                }
+            }
+            .lineLimit(1)
```

Comment:
also could possibly come from KMP constants 🤔

Reply:
Good point! I'll check with Algis about `.lineLimit(...)`

Reply:
So we came up whit idea to have a separated `.lineLimit(...)` for each label in list item. 👍